PROTOCOL.md

```diff
@@ -332,16 +332,16 @@ For example, streaming queries that are tailing the transaction log can use this
 
 The schema of the `add` action is as follows:
 
-Field Name | Data Type | Description
--|-|-
-path| String | A relative path to a data file from the root of the table or an absolute path to a file that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the data file path.
-partitionValues| Map[String, String] | A map from partition column to value for this logical file. See also [Partition Value Serialization](#Partition-Value-Serialization)
-size| Long | The size of this data file in bytes
-modificationTime | Long | The time this logical file was created, as milliseconds since the epoch
-dataChange | Boolean | When `false` the logical file must already be present in the table or the records in the added file must be contained in one or more `remove` actions in the same version
-stats | [Statistics Struct](#Per-file-Statistics) | Contains statistics (e.g., count, min/max values for columns) about the data in this logical file
-tags | Map[String, String] | Map containing metadata about this logical file
-deletionVector | [DeletionVectorDescriptor Struct](#Deletion-Vectors) | Either null (or absent in JSON) when no DV is associated with this data file, or a struct (described below) that contains necessary information about the DV that is part of this logical file.
+| Field Name       | Data Type                                            | Description                                                                                                                                                                                                                                                                            | optional/required |
```

Comment:
`Optional/Required` (capitalized)

Reply:
In https://github.com/delta-io/delta/blob/master/PROTOCOL.md#change-metadata we use lower-case, should I capitalize both?

Reply:
Cool. Then leave it lower-case. Thanks for checking!